app/js/handlebarsQuestionTable.js

```diff
@@ -0,0 +1,102 @@
+let baseUrl = getBaseUrl()
+let questionsTable = {data: []}
```

Comment:
Make this a function that returns the data format you need

Reply:
or just define it where you need it

Reply:
Turned into a function and used instead of variable